frontends/election-manager/src/screens/tally_report_screen.tsx

```diff
@@ -72,9 +75,15 @@ export function TallyReportScreen(): JSX.Element {
   assert(isAdminAuth(auth)); // TODO(auth) check permissions for viewing tally reports.
   const userRole = auth.user.role;
 
+  const location = useLocation();
+
   const { election } = electionDefinition;
   const statusPrefix = isOfficialResults ? 'Official' : 'Unofficial';
 
+  const castVoteRecordFileList = castVoteRecordFiles.fileList;
+  const hasCastVoteRecordFiles =
+    castVoteRecordFileList.length > 0 || !!castVoteRecordFiles.lastError;
```

Comment:
We repeat this logic enough, now, that it could go in a helper, maybe a method on the `CastVoteRecordFiles` class! https://github.com/votingworks/vxsuite/blob/f9d08a14cc68c1e0e5a31807d9487192c94512d9/frontends/election-manager/src/utils/cast_vote_record_files.ts#L94